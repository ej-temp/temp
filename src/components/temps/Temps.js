import React from 'react';
import PropTypes from 'prop-types';
import Temp from './Temp';
import { Link } from 'react-router-dom';

export default function Temps({ temps }) {
  const tempsArr = temps.map((temp, i) => {
    return (
      <Link to={`/${temp.id}`} key={i}>
        <li>
          <Temp name={temp.name} temp={temp.temp} />
        </li>
      </Link>
    );
  });

  return (
    <ul>
      {tempsArr}
    </ul>
  );
}

Temps.propTypes = {
  temps: PropTypes.array.isRequired
};
