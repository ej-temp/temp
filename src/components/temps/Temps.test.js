import React from 'react';
import { shallow } from 'enzyme';
import Temps from './Temps';

describe('Temps component', () => {
  it('renders Temps', () => {
    const wrapper = shallow(<Temps 
      temps={[]}
    />);
    expect(wrapper).toMatchSnapshot();
  });
});
