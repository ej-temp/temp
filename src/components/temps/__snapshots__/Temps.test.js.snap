// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Temps component renders Temps 1`] = `<ul />`;
